org.octave.Octave.json

```diff
@@ -0,0 +1,377 @@
+{
+  "app-id": "org.octave.Octave",
+  "runtime": "org.kde.Sdk",
+  "runtime-version": "5.9",
+  "sdk": "org.kde.Sdk",
+  "command": "octave",
+  "rename-icon": "octave",
+  "rename-desktop-file": "www.octave.org-octave.desktop",
+  "rename-appdata-file": "www.octave.org-octave.appdata.xml",
+  "finish-args": [
+    "--socket=wayland",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--share=network",
+    "--device=dri",
+    "--filesystem=home"
+  ],
+  "build-options": {
+    "cflags": "-O2",
+    "cxxflags": "-O2"
+  },
+  "modules": [
+    {
+      "name": "gcc-fortran",
+      "build-options": {
+        "ldflags": "-static-libgcc"
+      },
+      "config-opts": [
+        "--enable-languages=c,c++,fortran",
+        "--disable-multilib",
+        "--disable-bootstrap",
+        "--with-system-zlib",
+        "--enable-checking=release",
+        "--disable-stage1-checking",
+        "--disable-libsanitizer",
+        "--disable-libssp"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://mirrors.kernel.org/gnu/gcc/gcc-7.1.0/gcc-7.1.0.tar.gz",
+          "sha256": "3c187ef42e4cf657eec0ed641aac7f7e46261bcb1cea31b72dcf2dedddbbdea5"
+        },
+        {
+          "type": "shell",
+          "commands": [
+            "sed -e '/m64=/s/lib64/lib/' -i.orig gcc/config/i386/t-linux64"
+          ]
+        }
+      ]
+    },
+    {
+      "name": "openblas",
+      "no-autogen": true,
+      "build-options": {
+        "ldflags": "-L/app/lib64"
+      },
+      "make-args": [
+        "FC=gfortran",
+        "DYNAMIC_ARCH=1"
+      ],
+      "make-install-args": [
+        "PREFIX=/app"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/xianyi/OpenBLAS/archive/v0.2.20.tar.gz",
+          "sha256": "5ef38b15d9c652985774869efd548b8e3e972e1e99475c673b25537ed7bcf394"
+        }
+      ]
+    },
+    {
+      "name": "glu",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://mesa.freedesktop.org/archive/glu/glu-9.0.0.tar.bz2",
+          "sha256": "1f7ad0d379a722fcbd303aa5650c6d7d5544fde83196b42a73d1193568a4df12"
+        }
+      ],
+      "cleanup": [
```

Comment:
This should probably be global cleanup.

Reply:
These are the cleanup paths specified in the shared glu module. Since Octave is an IDE, there's not that much I can clean up. When installing packages or compiling C++ code using the Octave API with the mkoct command, a lot of octave's build dependencies are still needed.